<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>501</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/Resources/notepad.png</normaloff>:/image/Resources/notepad.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="CodeEditor" name="widgetEditer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>Search(&amp;S)</string>
    </property>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_S"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>LookUp help(&amp;H)</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/find.png</normaloff>:/image/Resources/find.png</iconset>
   </property>
   <property name="text">
    <string>Find(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/undo.png</normaloff>:/image/Resources/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/cut.png</normaloff>:/image/Resources/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/copy.png</normaloff>:/image/Resources/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/paste.png</normaloff>:/image/Resources/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select all(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/new.png</normaloff>:/image/Resources/new.png</iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/open.png</normaloff>:/image/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/save.png</normaloff>:/image/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/save_as.png</normaloff>:/image/Resources/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as(&amp;A)...</string>
   </property>
  </action>
  <action name="actionPageSetting">
   <property name="text">
    <string>页面设置(U)...</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/print.png</normaloff>:/image/Resources/print.png</iconset>
   </property>
   <property name="text">
    <string>Print(&amp;P)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit(&amp;X)</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Resources/redo.png</normaloff>:/image/Resources/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About TinyNotePad(&amp;A)</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font(&amp;F)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QWidget</extends>
   <header>codeeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
